plugins/cassandra/cassandrawu.cpp

```diff
@@ -1158,7 +1158,7 @@ static const CassandraXmlMapping wuGraphRunningMappings [] =
     {"partition", "int", NULL, hashRootNameColumnMapper},
     {"wuid", "text", NULL, rootNameColumnMapper},
     {"graphID", "text", NULL, stringColumnMapper},
-    {"subgraphID", "int", NULL, intColumnMapper},
+    {"subgraphID", "bigint", NULL, bigintColumnMapper},
```

Comment:
I'm following WUGraphIDType here. If we wanted to shrink it we could (as a separate Jira) but there's not a lot to lose leaving it as it is.